content/docs/04-clusters/06-cluster-management/15-node-pool.md

```diff
@@ -46,44 +46,44 @@ You can create a new node pool for an active cluster. To create a new node pool
-|**Disk Size**|Give the required storage size|
-
-### Worker Pool
-
-|**Parameter**     | **Description**|
-|------------------|---------------|
-|**Name**          |A descriptive name for the node pool.|
-|**Enable Autoscaler**|You can enable the autoscaler, by toggling the **Enable Autoscaler** button. Autoscaler scales up and down resources between the defined minimum and the maximum number of nodes to optimize resource utilization.|
-||Set the scaling limit by setting the **Minimum Size** and **Maximum Size**, as per the workload the number of nods will scale up from minimum set value to maximum set value and the scale down from maximum set value to minimum set value|
-|**Size**          |Number of VMs to be provisioned for the node pool.|
-|**Rolling Update**| Rolling update has two available options. Review the [Update Parameter](#update-parameter-table) table below for more details.
-|**[Labels](/clusters/cluster-management/taints#overviewonlabels)**|Add a label to apply placement constraints on a pod, such as a node eligible for receiving the workload.
-|**[Taints](/clusters/cluster-management/taints#overviewontaints)**|To set toleration to pods and allow (but do not require) the pods to schedule onto nodes with matching taints.|
-|**Instance type** |Select the compute instance type to be used for all nodes in the node pool.|
-|**Availability Zones**| Choose one or more availability zones. Palette provides fault tolerance to guard against hardware failures, network failures, etc., by provisioning nodes across availability zones if multiple zones are selected.|
-|**Disk Size**|Provide the required storage size
+
+### Master Node Pool
+
+| Property | Description |
+|-----------|-------------|
+| **Node pool name** | A descriptive name for the node pool. |
+| **Number of nodes in the pool** | Number of nodes to be provisioned for the node pool. For the master pool, this number can be 1, 3, or 5. |
+| **Allow worker capability** | Select this option to allow workloads to be provisioned on master nodes. |
+| **Additional labels** | Optional labels apply placement constraints on a pod. For example, you can add a label to make a node eligible to receive the workload. To learn more, refer to the [Overview on Labels](/clusters/cluster-management/taints#overviewonlabels). |
+| **Taints** | Sets toleration to pods and allows (but does not require) the pods to schedule onto nodes with matching taints. To learn more, refer to the [Overview on Taints](/clusters/cluster-management/taints#overviewontaints).|
+| **Cloud configuration** | Set the Resource Pool and Availability Zone in MAAS to select available servers from for deployment. Filter available servers to only those that have at least the amount of CPU and Memory selected. |
+| **Resource pool** | The MAAS resource pool from which to select available servers deployment.  |
```

Comment:
@ritawatson not all providers have this field. We need to keep this section as generic as possible. I noticed you used Maas in other  descriptions above.

Log into Palette prod, use the URL I gave you to review the different options so you can see what parameters each provider supports.